lib/custom-validators.js

```diff
@@ -18,10 +18,15 @@ const past = value => value === ''
 const since = (value, epoc) => value === ''
   || moment(value, /^\d{6,8}$/.test(value) ? 'DDMMYYYY' : 'DD/MM/YYYY').isSameOrAfter(moment(epoc).floor(24, 'hours'));
 
+const emailChars = function emailChars(value) {
+  return value === '' || this.regex(value, /^[a-z0-9._%+@-]+$/i);
+};
```

Comment:
Just because it's just a convention for them to be e-mail addresses. It's not really enforced.